vtds_cluster_kvm/private/config/config.yaml

```diff
@@ -273,9 +302,30 @@ cluster:
             # initialized. If this is null or not provided, the disk
             # will be an empty disk.
             source_image: null
+            # The source checksum block can be used to validate a source
+            # image. See the 'boot_disk' source information for more
+            # details.
+            source_checksum: {}
+            # The source checksum allows the source to be signed for
+            # security purposes. None specified here because no image
+            # was specified. If one were specified it would be a URL to
+            # a checksum file generated by the provider of the source
+            # image.
+            source_checksum: null
+            # The source checksum algorithm defaults to SHA256 but can
+            # be specified here. If the checksum file indicates a the
+            # actual algorithm used, that information is used and this
+            # setting is ignored. In cases where the file does not
+            # identify the algorithm, this is used. Supported options
+            # are MD5 and SHA256.
+            source_checksum_algorithm: SHA256
+            # Media type specifies what medium the disk image
+            # is. For non-boot disks, the only medium that is currently
+            # supported is qcow2.
+            media_type: qcow2
             # The 'disk_size_mb' field specifies the size in megabytes
             # (MB -- multiples of 1,000,000 bytes) of the disk.
```

Comment:
REVIEWERS: here I have trimmed back the additional disk (default) to 50GB to make it fit on more modest Virtual Blades.